diff --git a/node_modules/stream/index.js b/node_modules/stream/index.js
index 368bbfe..a0b12a2 100644
--- a/node_modules/stream/index.js
+++ b/node_modules/stream/index.js
@@ -19,7 +19,7 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-var Emitter = require('emitter');
+var Emitter = require('emitter-component');
 
 function Stream() {
   Emitter.call(this);
